pkg/test/istio.io/README.md

```diff
@@ -1,13 +1,42 @@
-# Istio.io Example Tests
+# Istio.io preliminary test cases
+
+Welcome to the development of the istio.io preliminary test cases!
+
+This work is experimental. This work is also under construction and may not work
+for you. The Istio developers have done some work to document known problems or
+workarounds. We expect this work will take several releases to complete.
 
 ## Overview
-This folder contains tests for the different examples on Istio.io.
+This folder contains tests for the different examples on [istio.io](https://istio.io).
+The goal of this work is to ensure [istio.io](https://istio.io) is tested according
+to the code in the [Istio repository](https://github.com/istio).
 
 ## Writing Tests
 
+This section needs fleshing out.
 
 ## Executing Tests Locally using Minikube
-* Create a Minikube environment as advised on istio.io
-* Run the tests as follows:
- go test ./... -p 1 --istio.test.env kube --istio.test.kube.config ~/.kube/config -v
 
+- Create a [Minikube environment](https://preliminary.istio.io/docs/setup/kubernetes/platform-setup/minikube/)
+  to develop the code.
+
+There has been some indication that the default minikube environment values of 16GB
+and 4 cores is not sufficient to run the test cases.  If after running the test cases,
+you see istio-system pods in the pending state, please attempt to debug the reason
+why they are pending so the Istio developers can improve these test cases.  Avaialbity
+of running the test cases locally is crucial to the progression of this work.
+
+- Change into the proper directory to run the test cases
+
+```bash
+$ cd $HOME/istio/pkg/test/istio.io
```

Comment:
this shouldn't be $HOME but rather $GOPATH/src/istio.io/istio/pkg/test/istio.io

Reply:
you should not have to be in the pkg/test/istio.io directory I think.. let me try it out locally

Reply:
`go test ./pkg/test/istio.io/... -p 1 --istio.test.env kube -v` works for me (other than failing with `error: the path "@samples/sleep/sleep.yaml@" does not exist`)

Reply:
I meant cd $ISTIO - which defined in our developer wiki is $GOPATH/src/istio.io.

@howardjohn ya, this is the same failure I get.  Maybe it is just temporary.

Reply:
@howardjohn - looks good, I will change the command to that operation.  go test . creates alot of failures, but the approach you recommended is simplest.

Reply:
I have a fix for @samples/sleep/sleep.yaml@. Will open that PR shortly.